static/app/views/settings/account/notifications/fields.tsx

```diff
@@ -88,6 +88,17 @@ export const ACCOUNT_NOTIFICATION_FIELDS: Record<string, FineTuneField> = {
       {value: '0', label: t('Off')},
     ],
   },
+  brokenMonitors: {
+    title: t('Broken Monitors'),
+    description: t(
+      'Notifications for monitors that have been in a failing state for a prolonged period of time'
+    ),
+    type: 'select',
+    options: [
+      {value: '1', label: t('On')},
+      {value: '0', label: t('Off')},
+    ],
```

Comment:
Should this be a select? I guess all the other ones are?

Reply:
yea just following the pattern we have going on in this file. I've been investigating these notification field definitions and think there is a decent amount of cleanup that can be done. For now just gonna stay consistent